src/els_compiler_diagnostics.erl

```diff
@@ -170,19 +172,32 @@ range(none) ->
 inclusion_range(IncludePath, Document) ->
   [Range|_] =
     inclusion_range(IncludePath, Document, include) ++
-    inclusion_range(IncludePath, Document, include_lib),
+    inclusion_range(IncludePath, Document, include_lib) ++
+    inclusion_range(IncludePath, Document, behaviour) ++
+    inclusion_range(IncludePath, Document, parse_transform),
   Range.
 
 -spec inclusion_range( string()
                      , els_dt_document:item()
-                     , include | include_lib) -> [poi_range()].
+                     , include | include_lib | behaviour | parse_transform)
+                     -> [poi_range()].
 inclusion_range(IncludePath, Document, include) ->
   POIs       = els_dt_document:pois(Document, [include]),
   IncludeId  = include_id(IncludePath),
   [Range || #{id := Id, range := Range} <- POIs, Id =:= IncludeId];
 inclusion_range(IncludePath, Document, include_lib) ->
   POIs       = els_dt_document:pois(Document, [include_lib]),
   IncludeId  = include_lib_id(IncludePath),
+  [Range || #{id := Id, range := Range} <- POIs, Id =:= IncludeId];
+inclusion_range(IncludePath, Document, behaviour) ->
+  POIs       = els_dt_document:pois(Document, [behaviour]),
+  %% IncludeId  = include_id(IncludePath),
```

Comment:
Any reason for the commented code? Also, these should be `BehaviourId` and `ParseTransformId`, since they are not related to _includes_ (probably a leftover from copy/paste).